content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/electric_dipole_definition/index.rst

```diff
@@ -0,0 +1,128 @@
+.. _definition_electric_dipole_index:
+
+Defining the Electrical Current Dipole
+======================================
+
+
+.. Purpose::
+
+    Here, we provide a physical description of the electrical current dipole.
+    This is used to develop a mathematical expression which can be used to replace the electrical source term in Maxwell's equations. 
+
+
+**General Definition**
+
+
+.. figure:: images/E_source_current_dipole.png
```

Comment:
Given the amount of stuff I have to work on, I may wait to embed the python script within the rst files. I also can't remember how quickly the script runs. I may have used many data points. I did put the scipts in Google Drive\GeoSciImages\Fundamental1\dipole_in_homogeneous